dashboard/app/models/course_offering.rb

```diff
@@ -332,7 +332,7 @@ def summarize_for_catalog(locale_code = 'en-us')
       professional_learning_program: professional_learning_program,
       video: video,
       published_date: published_date,
-      self_paced_pl_course_offering_path: self_paced_pl_course_offering&.path_to_latest_published_version(locale_code),
+      self_paced_pl_course_offering_path: get_self_paced_pl_course_offering_path(locale_code),
```

Comment:
why does this line need to change?

Reply:
to answer my own question -- it looks like you are working around the fact that `self_paced_pl_course_offering` no longer works. `self_paced_pl_course_offering` needs to work, by setting self_paced_pl_course_offering_id during seed

Reply:
Ok i think understand now!